tests/contrib/monitors/mixins/test_validation.py

```diff
@@ -0,0 +1,214 @@
+import pytest
+
+from spidermon.contrib.monitors.mixins import ValidationMonitorMixin
+from spidermon.contrib.scrapy.monitors import BaseScrapyMonitor
+from spidermon.data import Data
+
+
+stats = {
+    'spidermon/validation/fields': 100,
+    'spidermon/validation/fields/errors': 20,
+    'spidermon/validation/fields/errors/missing_required_field': 15,
+    'spidermon/validation/fields/errors/missing_required_field/field2': 5,
+    'spidermon/validation/fields/errors/missing_required_field/field3': 10,
+    'spidermon/validation/fields/errors/': 10,
+    'spidermon/validation/items': 10,
+}
+
+
+class DummyValidationMonitor(BaseScrapyMonitor, ValidationMonitorMixin):
+    def __init__(self, stats, methodName="runTest", name=None):
+        super(DummyValidationMonitor, self).__init__(methodName, name)
+        self.data = Data({'stats': stats})
+        self.new_behavior = True
+
+    def runTest(self):
+        pass
+
+
```

Comment:
I think that at least one test with `allowed_count > 0` to check the threshold message would be nice